lib/themes/dosomething/paraneue_dosomething/js/lab.js

```diff
@@ -0,0 +1,24 @@
+import Onboarding from './components/Onboarding';
+import ContextSlide from './components/ContextSlide';
+
+const $ = require('jquery');
+const React = require('react');
+const ReactDom = require('react-dom');
```

Comment:
yeah, I hear ya. I've forced myself to like it for the sake of simplicity, and consistency in other projects 😖